server/api/routes/meeting.cjs

```diff
@@ -0,0 +1,129 @@
+const router = require('express').Router();
+const { Meeting, Message, Rating } = require('../../db/index.cjs');
+const { requireToken, isAdmin } = require('../authMiddleware.cjs');
+
+// Don't belive needs middleware it'd be the server creating the meeting when needed
+router.post('/', async (req, res, next) => {
+  try {
+    console.log(req.body.userId);
+    const newMeeting = await Meeting.create(req.body);
+    res.status(200).json(newMeeting);
+  } catch (err) {
+    next(err);
+  }
+});
+// same case with above
+router.put('/:meetingId', async (req, res, next) => {
+  try {
+    const meeting = await Meeting.findByPk(req.params.meetingId);
+
+    if (meeting) {
+      res.json(await Meeting.update(req.body));
```

Comment:
check for what is being passed in the body to make sure only the correct info is being changed